ds/beis_indicators/utils/dir_file_management.py

```diff
@@ -101,8 +95,17 @@ def make_indicator(table,target_path,var_lookup,year_var,nuts_var='nuts_code',nu
     t = t[['year','nuts_id','nuts_year_spec',var_code]]
     
     print(t.head())
-    
-    #Save in the processed folder
-    t.to_csv(f'{project_dir}/data/processed/{target_path}/{var_code}.csv',index=False)
-    
-    
\ No newline at end of file
+
+    return(t)
+
+def save_indicator(table,target_path,var_name):
+    '''
+    Function to save an indicator
+
+    Args:
+        table (pandas.DataFrame)
+        path (str)
+        var_name (variable name)
+
+    '''
+    table.to_csv(f'{project_dir}/data/processed/{target_path}/{var_name}.csv',index=False)    
```

Comment:
I woudn't use a global variable in a function, but I'd rather pass it as an argument